src/app/tests/suites/commands/system/SystemCommands.cpp

```diff
@@ -68,12 +69,7 @@ CHIP_ERROR SystemCommands::Start(const char * identity, const chip::app::Cluster
 
     if (value.kvs.HasValue())
     {
-        VerifyOrReturnError(value.kvs.Value().size() < 128, CHIP_ERROR_INVALID_ARGUMENT);
-        builder.Add(" --KVS ");
-        char kvs[128];
-        memset(kvs, '\0', sizeof(kvs));
-        strncpy(kvs, value.kvs.Value().data(), value.kvs.Value().size());
-        builder.Add(kvs);
+        AddSystemCommandArgument(builder, "--KVS", value.kvs.Value());
```

Comment:
Suggest renaming `--KVS` everywhere with `--storage_location`. Calling it KVS is a bit of a misnomer and is also jargon

Reply:
So `--KVS` is the command line option for all Linux application. This really is just setting it up to pass directly to the Linux application.

Reply:
To be clear, to change this value to `--storage_location`, we would need to modify this outside of test first.